Tests/ValidationTests.swift

```diff
@@ -247,6 +247,78 @@ final class ContentTypeValidationTestCase: BaseTestCase {
         }
     }
 
+    func testThatValidationForRequestWithUnacceptableContentTypeResponseFailsWhenMultipleContentTypesAreAcceptable() {
```

Comment:
This test title doesn't really make sense, since this test is about ensuring the possible content types are sorted, not that the failure happens in the first place. I'd suggest `testThatContentTypeValidationFailureSortsPossibleContentTypes()`.